Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -239,23 +248,34 @@ void ITSClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
           hGroupedClusterSizeLayerSummary[lay]->Fill(npix);
         }
       }
+
+      // Transformation to the local --> global
+      mGeom->fillMatrixCache(o2::math_utils::bit2Mask(o2::math_utils::TransformType::L2G));
+      auto gloC = mGeom->getMatrixL2G(ChipID) * locC;
+      float phi = (float)TMath::ATan2(gloC.Y(), gloC.X());
+
+      phi = (float)(phi * 180 / TMath::Pi());
+      hAverageClusterOccupancySummaryZPhi[lay]->Fill(gloC.Z(), phi);
+      hAverageClusterSizeSummaryZPhi[lay]->Fill(gloC.Z(), phi, (float)npix);
+
+      hAverageClusterOccupancySummaryCoarse[lay]->Fill(getHorizontalBin(locC.Z(), chip, lay, lane), getVerticalBin(locC.X(), sta, lay));
+      hAverageClusterSizeSummaryCoarse[lay]->Fill(getHorizontalBin(locC.Z(), chip, lay, lane), getVerticalBin(locC.X(), sta, lay), (float)npix);
```

Comment:
I would add also a `SetBinError` for these 4 guys (see the others TH2 with the bit `kisAverage` set). So far we did not care about the error, hence we set it to `1e-15` (a random small value). This is important for the QC merger that in this way will make an average of the plot "without" taking the bin errors as weights.

Reply:
Done